t/greppable.t

```diff
@@ -35,6 +35,9 @@ $t.test-gist(‘“…” is added to long paths’,
 $t.test-gist(‘“…” is not added to root files’,
              %(‘result.md’ => none /‘``…/README.md``’/));
 
+$t.test(‘single line/module returned’,
+      “{$t.bot-nick}: 6lang”,
```

Comment:
This is fine but fragile, I'd go with something like:

> \<AlexDaniel\> greppable: ought to cover the same functionality as this class, maybe long-term we
> \<greppable6\> AlexDaniel, 1 lines, 1 modules: https://gist.github.com/fafdd7e108cdd382ddf0d7a536f80385

This is pretty much guaranteed to always return one module.